src/main/java/frc/robot/commands/auto/Auto.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot.commands.auto;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.Filesystem;
+import edu.wpi.first.wpilibj.controller.PIDController;
+import edu.wpi.first.wpilibj.controller.RamseteController;
+import edu.wpi.first.wpilibj.kinematics.DifferentialDriveKinematics;
+import edu.wpi.first.wpilibj.trajectory.Trajectory;
+import edu.wpi.first.wpilibj.trajectory.TrajectoryUtil;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.RamseteCommand;
+import edu.wpi.first.wpilibj.controller.SimpleMotorFeedforward;
+import edu.wpi.first.wpilibj2.command.*;
+import frc.robot.CONSTANTS;
+import frc.robot.subsystems.DriveTrain;
+
+public class Auto extends SequentialCommandGroup {
+
+    DriveTrain driveTrain;
```

Comment:
Generally avoid the default visibility specifier unless you specifically need it, 'private' here is fine.

Reply:
Or 'protected', if you want this accessible in sub classes.